### PR TITLE
Add organ types RvA, KKK, and AVW

### PR DESCRIPTION
**Pull request by MaxOpperman**
Dear Web Committee,

Could you please revise this addition of organ types. I would like to see the Advisory Board, Audit Committee, and GM Task Force types on the website. I also made a pull request for the GEWIS database.
I'm looking forward to your response.

Kind regards,
Max Opperman

### CODE REVIEW

File: module/Application/language/nl.po [hunk 1]
```diff
@@ -1957,6 +1957,18 @@ msgstr "AV Commissie"
...
+msgid "GM Task Force"
+msgstr "AV Werkgroep"
+
+#: module/Decision/view/decision/organ/index.phtml:41
+msgid "Audit Committee"
+msgstr "KKK"
+
+#: module/Decision/view/decision/organ/index.phtml:44
+msgid "Advisory Board"
+msgstr "RvA"
```
Comment (Koen1999) [inline module/Application/language/nl.po:1970]: Are you sure you prefer an abbreviation here?

**Review by Koen1999 — COMMENTED** [review]
Overall the changes look good.

File: module/Decision/src/Decision/Model/SubDecision/Foundation.php [hunk 2]
```diff
@@ -17,8 +17,11 @@ class Foundation extends SubDecision
 {
 
     const ORGAN_TYPE_COMMITTEE = 'committee';
-    const ORGAN_TYPE_AV_COMMITTEE = 'avc';
```
Comment (kokx) [inline module/Decision/src/Decision/Model/SubDecision/Foundation.php:20]: In the gewisdb PR, this is left as is (as in its left at `ORGAN_TYPE_AV_COMMITTEE`). So either change it there as well, or do not change it here. But keep it consistent.

**Review by kokx — CHANGES_REQUESTED** [review]
Looks pretty good. We should merge this before the PR on the gewisdb gets merged. But first fix the consistency.

File: module/Decision/src/Decision/Model/SubDecision/Foundation.php (hunk 2 above)
Comment (MaxOpperman) [reply module/Decision/src/Decision/Model/SubDecision/Foundation.php:20]: Fixed this in the gewisdb PR

**Review by Koen1999 — APPROVED** [review]
I have tested the changes in combination with the changes to gewisdb and it works as expected.

**Pull request merged**